lib/std/crypto/aes.zig

```diff
@@ -3,249 +3,46 @@
 // This file is part of [zig](https://ziglang.org/), which is MIT licensed.
 // The MIT license requires this copyright notice to be included in all copies
 // and substantial portions of the software.
-// Based on Go stdlib implementation
 
 const std = @import("../std.zig");
-const mem = std.mem;
 const testing = std.testing;
+const builtin = std.builtin;
+
+pub const aes = struct {
```

Comment:
May as well flatten this out, since the file is already aes.zig and is therefore a struct named aes. Basically, delete this line and the corresponding line with `};` and then unindent everything, and then the `@import` can change like this:

```diff
-const aes = @import("aes.zig").aes;
+const aes = @import("aes.zig");
```

Reply:
Done.